src/markdown/centering/micromark-centering.js

```diff
@@ -0,0 +1,109 @@
+import assert from 'assert';
+import markdownSpace from 'micromark/dist/character/markdown-space';
+import markdownLineEnding from 'micromark/dist/character/markdown-line-ending';
+import spaceFactory from 'micromark/dist/tokenize/factory-space';
+import types from 'micromark/dist/constant/types';
+import codes from 'micromark/dist/character/codes';
+import { shallowEqual } from 'markdown/utils';
+
+function tokenizeCentering(effects, ok, nok) {
+  let size = 0;
```

Comment:
Should this get reset to 0 at some point?

Reply:
It doesn't have to. The tokenize function is called again when a new block is attempted to be parsed.